packages/puppeteer-extra-plugin-devtools/readme.md

```diff
@@ -74,6 +74,7 @@ Type: `function (opts)`
         -   `opts.auth.user` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?** Username (default: 'user')
         -   `opts.auth.pass` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?** Password (will be generated if not provided)
     -   `opts.prefix` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)?** The prefix to use for the localtunnel.me subdomain (default: 'devtools-tunnel')
+    -   `opts.localtunnel` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)?** Advanced options to pass to [localtunnel](https://github.com/localtunnel/localtunnel#options)
```

Comment:
I had to add this line manually cause my `Prettier` would attempt to re-format the whole document. It could be a good idea to re-lint & re-prettify the whole repository, some rules may have been updated meanwhile.